jobs/tox.ini

```diff
@@ -5,9 +5,13 @@ skip_missing_interpreters = True
 [testenv]
 usedevelop=true
 commands =
+    python setup.py develop
```

Comment:
In Docker, to install `buildhub` itself, I used `pip install -e jobs` which effectively executes the `setup.py`. But I don't know how to do that in tox.

Reply:
I never was a tox expert, and can't help much :( But I can't remember having to do this though

Reply:
With tox your can use `usedevelop = true`